.markdownlint.json

```
@@ -26,16 +26,18 @@
     "no-hard-tabs": true,
     "no-inline-html": {
         "allowed_elements": [
-            "a",
-            "br",
-            "font",
-            "kbd",
-            "p",
-            "sup"
+            "h1", "h2", "h3", "h4", "h5", "h6",
+            "p", "div", "blockquote",
+            "pre",
+            "b", "i", "strong", "em", "tt", "code", "ins", "del", "sup", "sub", "kbd", "samp", "q", "var",
```

We should probably conform to 1 way of italicizing and bolding. I recommend using `em` and `strong`.

```suggestion
            "strong", "em", "tt", "code", "ins", "del", "sup", "sub", "kbd", "samp", "q", "var",
```